tools/Azure.Mcp.Tools.Aks/src/AksSetup.cs

```diff
@@ -18,13 +17,17 @@ public class AksSetup : IAreaSetup
     public void ConfigureServices(IServiceCollection services)
     {
         services.AddSingleton<IAksService, AksService>();
+
+        services.AddSingleton<ClusterListCommand>();
+        services.AddSingleton<ClusterGetCommand>();
+        services.AddSingleton<NodepoolListCommand>();
+        services.AddSingleton<NodepoolGetCommand>();
```

Comment:
This might be a bad pattern, but is there a cleaner way to go about adding the individual commands?

Nothing major, just feels that we have to repeat similar work in both `ConfigureServices` and `RegisterCommands`, where each command has to be declared in both.

Reply:
I described the question poorly, let me try to ask it in a different way.

Would `ConfigureServices` ever be called and `RegisterCommands` not be called?

What I meant is both methods have some overlap where you have to do something with `ClusterListCommand` and if it's possible it'd be nice to just need to do just one thing with `ClusterListCommand`. AKS here isn't a great example, but larger areas have many, many tools you need to make sure you setup properly.